pkg/dss/auth/auth.go

```diff
@@ -42,6 +42,14 @@ func (m *missingScopesError) Error() string {
 	return strings.Join(m.s, ", ")
 }
 
+type tokenExpireError struct {
```

Comment:
nit: how about 2 vars: tokenExpired, and tokenExpiredTooFar, ie:

tokenExpired = errors.New("the token is already expired")